docs/develop_guide/contributing.rst

```diff
@@ -0,0 +1 @@
+.. mdinclude:: ../../CONTRIBUTING.md
```

Comment:
There isn't a contributing.md in the repo so this line causes a few errors when making the documentation.